### PR TITLE
fix: eth network balance fetching

### PR DESCRIPTION
**Pull request by noahjoeris**
- fixes the error of unsuccessfully fetching token balances on ethereum network

### CODE REVIEW

**Review by Victor-Salomon — APPROVED** [review]
nice and simple fix 😎

File: app/src/registry/index.ts [hunk 1]
```diff
@@ -93,7 +95,6 @@ export function snowbridgeWrapped(): Origin {
 
 // Hack - importing this after types definition to avoid circular dependency and import timing issues
 import * as Mainnet from './mainnet'
-export * as Mainnet from './mainnet'
```
Comment (NunoAlexandre) [inline app/src/registry/index.ts:96]: Can we undo this change?

File: app/src/registry/index.ts [hunk 2]
```diff
@@ -46,6 +46,8 @@ export const SNOWBRIDGE_MAINNET_PARACHAIN_URLS = [
 ]
 
 export function getNativeToken(chain: Chain): Token {
+  if (chain.network === 'Ethereum') return Mainnet.Eth.ETH
```
Comment (NunoAlexandre) [inline app/src/registry/index.ts:49]: 👌🏻

File: app/src/registry/index.ts (hunk 1 above)
Comment (NunoAlexandre) [reply app/src/registry/index.ts:96]: What command do you run? I'm running lint and format and it doesn't do that and frankly I don't quite like the change because it moves the export away :/ but it's not important

**Review by NunoAlexandre — APPROVED** [review]
🙌🏻
